.ci.yaml

```diff
@@ -3750,6 +3750,25 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac channels_integration_test
```

Comment:
Is this task expected to run on either x64 or arm64?
/cc @jmagman is working on distinguishing these Mac host only tasks: https://github.com/flutter/flutter/pull/109889

Reply:
It's best if it runs on both but not necessary.  Otherwise it doesn't matter which one.